RijsMuseum/.idea/gradle.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All files from .idea folder should be ignored. Check how to ignore whole folder and add it to your .gitignore file